api/src/Entity/Comment.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Doctrine\Orm\Filter\SearchFilter;
+use ApiPlatform\Metadata\ApiFilter;
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Delete;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\Link;
+use ApiPlatform\Metadata\Post;
+use App\InputFilter;
+use App\Repository\CommentRepository;
+use App\State\CommentCreateProcessor;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * A Comment.
```

Comment:
Since this will end up in the OpenAPI docs, can we be a bit more verbose here? E.g.
```suggestion
 * A Comment someone left on an activity, to give feedback on the planned programme,
 * for notes which are only relevant during camp planning, or for other communication.
```